build.gradle.kts

```diff
@@ -49,19 +49,6 @@ buildscript {
         gradlePluginPortal()
         maven("https://maven.pkg.jetbrains.space/kotlin/p/kotlin/dev")
     }
-
-    val kotlin_version: String by extra
-    val atomicfu_version: String by extra
-    val validator_version: String by extra
-
-    dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
-        classpath("org.jetbrains.kotlinx:atomicfu-gradle-plugin:$atomicfu_version")
-        classpath("org.jetbrains.kotlin:kotlin-serialization:$kotlin_version")
-        classpath("org.jetbrains.kotlinx:binary-compatibility-validator:$validator_version")
-    }
-
-    CacheRedirector.configureBuildScript(rootProject, this)
```

Comment:
Why don't we need atomicfu/serialization and stuff now?

Reply:
Because we replaced it with version catalog